portfolio/src/main/webapp/script.js

```diff
@@ -78,3 +85,25 @@ function getGallery() {
     }
     wrapper.appendChild(imageContainer);
 }
+
+/**
+ * Fetches comments from the server and adds them to the DOM.
+ */
+function getComments() {
+    const wrapper = document.getElementById("wrapper");
+    wrapper.textContent = '';
+    fetch('/data').then(response => response.json()).then((comments) => {
+        for (let comment of comments) {
+            wrapper.appendChild(
+                createListElement(comment.text + ', by ' + comment.author +
+                                  ', posted on: ' + comment.date));
```

Comment:
The `<li>` tags should be used inside a list, so you would need an `<ol>` or `<ul>` tag which holds the list elements.